easybuild/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-11.2.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PerlModule'
+
+name = 'Bio-DB-HTS'
+version = '3.01'
+
+homepage = 'https://metacpan.org/release/Bio-DB-HTS'
+description = "Read files using HTSlib including BAM/CRAM, Tabix and BCF database files"
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+source_urls = ['https://cpan.metacpan.org/authors/id/A/AV/AVULLO/']
+sources = ['Bio-DB-HTS-%(version)s.tar.gz']
+checksums = ['12a6bc1f579513cac8b9167cce4e363655cc8eba26b7d9fe1170dfe95e044f42']
+
+builddependencies = [('pkg-config', '0.29.2')]
+
+dependencies = [
+    ('Perl', '5.34.0'),
+    ('BioPerl', '1.7.8'),
+    ('HTSlib', '1.14'),
+]
+
```

Comment:
`preconfigopts = "env HTSLIB_DIR=$EBROOTHTSLIB"` is no longer necessary?

Reply:
@migueldiascosta could you say why? What has changed?

Reply:
@migueldiascosta could you say why? What has changed?

Reply:
@verdurin I was asking a question :) but I see that it has been addressed in #15116, and this PR no longer needs to add this easyconfig?

Reply:
Ah, no, #15116 also includes `Bio-DB-HTS-3.01-GCC-11.2.0.eb`, so there's a merge conflict here. I'll get that fixed then...